s32/mcux/devices/S32Z270/S32Z270_glue_mcux.h

```diff
@@ -247,4 +247,49 @@
+#define DMA3_0_ERROR_IRQS                           { RTU_DMA0_ERR_IRQn }
+
+/** Peripheral EDMA3_1 base address */
+#define EDMA3_1_BASE                                IP_EDMA_1_MP_BASE
+/** Peripheral EDMA3_1 base pointer */
+#define EDMA3_1                                     ((DMA_Type *)EDMA3_1_BASE)
+/** Interrupt vectors for the DMA3_1 peripheral type */
+#define DMA3_1_IRQS                                 { RTU_DMA1_0_IRQn, RTU_DMA1_1_IRQn, RTU_DMA1_2_IRQn, RTU_DMA1_3_IRQn, RTU_DMA1_4_IRQn, RTU_DMA1_5_IRQn, RTU_DMA1_6_IRQn, RTU_DMA1_7_IRQn, RTU_DMA1_8_IRQn, RTU_DMA1_9_IRQn, RTU_DMA1_10_IRQn, RTU_DMA1_11_IRQn, RTU_DMA1_12_IRQn, RTU_DMA1_13_IRQn, RTU_DMA1_14_IRQn, RTU_DMA1_15_IRQn }
+#define DMA3_1_ERROR_IRQS                           { RTU_DMA1_ERR_IRQn }
+
+/** Peripheral EDMA3_4 base address */
+#define EDMA3_4_BASE                                IP_EDMA_4_MP_BASE
+/** Peripheral EDMA3_4 base pointer */
+#define EDMA3_4                                     ((DMA_Type *)EDMA3_4_BASE)
+/** Interrupt vectors for the DMA3_4 peripheral type */
+#define DMA3_4_IRQS                                 { RTU_DMA4_0_IRQn, RTU_DMA4_1_IRQn, RTU_DMA4_2_IRQn, RTU_DMA4_3_IRQn, RTU_DMA4_4_IRQn, RTU_DMA4_5_IRQn, RTU_DMA4_6_IRQn, RTU_DMA4_7_IRQn, RTU_DMA4_8_IRQn, RTU_DMA4_9_IRQn, RTU_DMA4_10_IRQn, RTU_DMA4_11_IRQn, RTU_DMA4_12_IRQn, RTU_DMA4_13_IRQn, RTU_DMA4_14_IRQn, RTU_DMA4_15_IRQn }
+#define DMA3_4_ERROR_IRQS                           { RTU_DMA4_ERR_IRQn }
+
+/** Peripheral EDMA3_5 base address */
+#define EDMA3_5_BASE                                IP_EDMA_5_MP_BASE
+/** Peripheral EDMA3_5 base pointer */
+#define EDMA3_5                                     ((DMA_Type *)EDMA3_5_BASE)
+/** Interrupt vectors for the DMA3_5 peripheral type */
+#define DMA3_5_IRQS                                 { RTU_DMA5_0_IRQn, RTU_DMA5_1_IRQn, RTU_DMA5_2_IRQn, RTU_DMA5_3_IRQn, RTU_DMA5_4_IRQn, RTU_DMA5_5_IRQn, RTU_DMA5_6_IRQn, RTU_DMA5_7_IRQn, RTU_DMA5_8_IRQn, RTU_DMA5_9_IRQn, RTU_DMA5_10_IRQn, RTU_DMA5_11_IRQn, RTU_DMA5_12_IRQn, RTU_DMA5_13_IRQn, RTU_DMA5_14_IRQn, RTU_DMA5_15_IRQn }
+#define DMA3_5_ERROR_IRQS                           { RTU_DMA5_ERR_IRQn }
+
+/** Peripheral DMA3 base address */
+#define DMA3_BASE                                   {EDMA3_0_BASE, EDMA3_1_BASE, EDMA3_4_BASE, EDMA3_5_BASE}
+/** Peripheral DMA3 base pointer */
+#define DMA3                                        {EDMA3_0, EDMA3_1, EDMA3_4, EDMA3_5}
```

Comment:
Hmm, i'm getting lost here what you define, is the `DMA3` here same as `DMA_BASE_PTRS`; `DMA3_BASE` and `DMA_BASE_ADDRS`. Also i see there is expression like:
`#define FSL_FEATURE_EDMA_INSTANCE_HAS_TRANSFER_MODEn(x) (((x) == DMA3) ? (0) : (-1))`
It is seems an invalid expression (comparing a number with an array?). Pls refine the definition in here

Reply:
For the first point, DMA3 and DMA3_BASE are the same with DMA_BASE_PTRS and DMA_BASE_ADDRS, so I removed it.
And the second point, I updated the definition.